src/io_scene_vrm/importer/vrm_animation_importer.py

```diff
@@ -349,16 +362,16 @@ def import_vrm_animation(context: Context, path: Path, armature: Object) -> set[
     expression_name_to_translation_keyframes: dict[
         str, tuple[tuple[float, Vector], ...]
     ] = {}
-    for expression_name, node_index in expression_name_to_node_index.items():
-        if not 0 <= node_index < len(node_dicts):
+    for expression_name, node_idx in expression_name_to_node_index.items():
+        if not 0 <= node_idx < len(node_dicts):
             continue
-        node_dict = node_dicts[node_index]
-        if not isinstance(node_dict, dict):
+        node_dict_ = node_dicts[node_idx]
+        if not isinstance(node_dict_, dict):
             continue
 
-        translation = node_dict.get("translation")
+        translation = node_dict_.get("translation")
         if isinstance(translation, list) and translation:
-            default_preview_value = translation[0]  # TODO: Matrixだった場合
+            default_preview_value = translation[0]
```

Comment:
This Japanese means that glTF may also express the position of nodes using a matrix, so we will support that in the future. Please do not delete it, but leave it as it is.

Or, please send us a pull request to translate it into English.